frontend/src/core/hotkeys/hotkeys.ts

```diff
@@ -127,6 +127,11 @@ const DEFAULT_HOT_KEY = {
     group: "Editing",
     key: "Mod-f",
   },
+  "cell.selectNextOccurrence": {
+    name: "Select next occurrence",
+    group: "Editing",
+    key: "Mod-d",
```

Comment:
For some reason, this doesn't work for me (it just bookmarks the page on Chrome, using Linux). Not a huge deal, just noting.

Reply:
this is my favorite hotkey in vscode. i was glad to see codemirror has it. did you have some text selected? and in an editor?